fe/src/main/java/org/apache/doris/analysis/ColumnDef.java

```diff
@@ -131,6 +131,16 @@ public void analyze(boolean isOlap) throws AnalysisException {
 
         Type type = typeDef.getType();
 
+        // disable Bitmap Hll type in keys, values without aggregate function.
+        if ( type.isBitmapType() || type.isHllType() ){
```

Comment:
Should keep code style with original code.
```suggestion
        if (type.isBitmapType() || type.isHllType()) {
```